src/Parallel/Command/WorkerCommandLineFactory.php

```diff
@@ -41,8 +41,15 @@ public function create(
         int $port
     ): string {
         $commandArguments = array_slice($_SERVER['argv'], 1);
-        $args = [PHP_BINARY, $mainScript, ...$commandArguments];
 
+        // add implicit "process" command name if missing
+        if ($commandArguments !== [] && ($commandArguments[0] !== 'process' && $commandArguments[0] !== 'p') && ! defined(
```

Comment:
this should be verified with `--help` or `--version` as well, which doesn't process anything, just show the info.

Reply:
I've added those 2 to ci :+1: